### ISSUE TITLE
[BUG] PIP 20.2 Dependency Resolver

### ISSUE THREAD

**Issue by greenfrogs** [uclapi/uclapi#2905]
Pip is introducing a new dependency resolver in 20.2 this causes a problem with the vine and amqp packages, this can be tested by running:
```
pip install -r requirements.txt --use-feature=2020-resolver
```

```
ERROR: Cannot install vine==1.3.0 and amqp 5.0.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested vine==1.3.0
    amqp 5.0.1 depends on vine==5.0.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by greenfrogs** [comment]
A possible fix is to upgrade `vine==1.3.0` to `vine==5.0.0`, however I am not sure if this will cause issues somewhere else.

**Comment by Feverfew** [comment]
#2930 and your suggestion to bump to `5.0.0` will resolve this issue according to this commit
https://github.com/celery/celery/commit/8d6f7a8a787c0ffe2f79ef877e52bb81da335890
